### PR TITLE
docs: ✏️ Connect Boundary UI to Boundary local instance

### PR DESCRIPTION
**Pull request by calcaide**
Add documentation on how to connect Boundary UI, admin and desktop
version, to a Boundary local instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
 Assets are saved to `ui/desktop/electron-app/out/make/`.
 DMG packaged desktop UI is available at asset location as `boundary.dmg`.
 
+### Connect Boundary UI to Boundary local instance
+
+This describes how to connect local Boundary UI to your local instance of Boundary.
+
+This assumes you already have up and running a boundary instance in your local environment with a listener `127.0.0.1:9200` (default behaviour). If you do not have it, please [follow Boundary documentation](https://github.com/hashicorp/boundary#build-and-start-boundary-in-dev-mode) for it.
```
Comment (randallmorey) [inline README.md:135]: "a Boundary dev instance"

Comment (randallmorey) [reply README.md:135]: Maybe change the second sentence to be all one link "Learn how to start a local Boundary dev instance"?

File: README.md [hunk 2]
```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
...
+
+Following terminal instructions, open in your browser: `http://localhost:4200/`.
+
+Once you open the UI you will see the login screen.
+
+By default, for development purposes, credentials are:
+- Login name: `admin`.
+- Password: `password`
+
+*Be aware these are default credentials, if you change them in Boundary, use the ones you set before*
```
Comment (randallmorey) [inline README.md:150]: Add a space after this line.

File: README.md [hunk 3]
```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
...
+#### For desktop
+
+This assumes you are within `boundary-ui/ui/desktop`.
+
+You will need to turn `off` mirage and run it: `$ ENABLE_MIRAGE=false yarn start:desktop`.
+
+You will see the login screen.
+
+Make sure the origin URL is pointing `http://localhost:9200`.
```
Comment (randallmorey) [inline README.md:159]: I might state this along the lines of "enter the origin of your Boundary dev instance" and mention that the default is localhost:9200.

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
...
+- Password: `password`
+
+*Be aware these are default credentials, if you change them in Boundary, use the ones you set before*
+#### For desktop
+
+This assumes you are within `boundary-ui/ui/desktop`.
+
+You will need to turn `off` mirage and run it: `$ ENABLE_MIRAGE=false yarn start:desktop`.
+
+You will see the login screen.
```
Comment (gsusmi) [inline README.md:157]: Could eliminate this line by appending `In login screen, ...` to next line as running previous step would launch desktop automatically.

File: README.md [hunk 5]
```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
...
+
+Make sure the origin URL is pointing `http://localhost:9200`.
+
+By default, for development purposes, credentials are:
+- Login name: `admin`.
+- Password: `password`
```
Comment (gsusmi) [inline README.md:163]: See note above about embedding credentials.

**Review by gsusmi — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -125,6 +128,42 @@ environment variable to set signing certificate name (e.g Developer ID Applicati
...
+
+You will need to turn `off` mirage, tell the UI where to find Boundary and run it: `$ ENABLE_MIRAGE=false API_HOST=http://localhost:9200 yarn start`.
+
+Following terminal instructions, open in your browser: `http://localhost:4200/`.
+
+Once you open the UI you will see the login screen.
+
+By default, for development purposes, credentials are:
+- Login name: `admin`.
+- Password: `password`
```
[the remark this answers is not in the document]
Comment (calcaide) [reply README.md]: Agreed. I delete the credentials and when with your suggestion. Thanks!

File: README.md (hunk 5 above)
Comment (calcaide) [reply README.md]: Indeed. Changed.

File: README.md (hunk 2 above)
Comment (calcaide) [reply README.md]: I deleted credentials as per suggestion. So this line does not exist anymore.

File: README.md (hunk 3 above)
Comment (calcaide) [reply README.md]: Updated, thanks for the suggestion!

File: README.md (hunk 4 above)
Comment (calcaide) [reply README.md]: You are right. Updated.

**Review by gsusmi — APPROVED** [review]

**Pull request merged**